pipeline_steps/common.groovy

```diff
@@ -1115,4 +1136,24 @@ Boolean isUserAbortedBuild() {
   return userAborted
 }
 
+String genDockerBuildArgs() {
+  return sh(script: """#!/usr/bin/env python3
+from os import environ
+from shlex import quote
+
+# We don't want to set a default for build_args as these do not always apply
+build_args = environ.get('SLAVE_CONTAINER_DOCKERFILE_BUILD_ARGS')
+dockerfile = environ.get('SLAVE_CONTAINER_DOCKERFILE_PATH', './Dockerfile');
```

Comment:
I've re-added this default back in, but it can be removed once this merges since the live lint jobs etc. will have that parameter set and won't need a default here.